<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pwdDialog</class>
 <widget class="QDialog" name="pwdDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>85</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Şifre Girişi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/resources/Weather-icon.png</normaloff>:/resources/Weather-icon.png</iconset>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tamam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>pwdDialog</receiver>
   <slot>acceptButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>pwdDialog</receiver>
   <slot>cancelButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>pwdDialog</receiver>
   <slot>acceptButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>acceptButton()</slot>
  <slot>cancelButton()</slot>
 </slots>
</ui>
